include/z64cutscene_commands.h

```diff
@@ -11,445 +11,274 @@
-#define CS_CAM_EYE_REL_TO_PLAYER(continueFlag, roll, frame, viewAngle, xPos, yPos, zPos, unused) \
-    CMD_BBH(continueFlag, roll, frame), CMD_F(viewAngle), CMD_HH(xPos, yPos), CMD_HH(zPos, unused)
-
-/**
- * ARGS
- *   s16 startFrame (s), s16 endFrame (e)
- * FORMAT
- *   Capital U is Unused , may be consistently zero
- *   00000006 0001ssss eeee0000
- *   size = 0xC
- */
-#define CS_CAM_FOCUS_POINT_PLAYER_LIST CS_CAM_AT_REL_TO_PLAYER_LIST
-#define CS_CAM_AT_REL_TO_PLAYER_LIST(startFrame, endFrame) \
-    CS_CMD_CAM_AT_REL_TO_PLAYER, CMD_HH(0x0001, startFrame), CMD_HH(endFrame, 0x0000)
-/**
- * ARGS
- *   s8 continueFlag (c), s8 roll (r), s16 frame (f), f32 viewAngle (a),
- *   s16 xPos (x), s16 yPos (y), s16 zPos (z)
- * FORMAT
- *   Capital U is Unused
- *   ccrrffff aaaaaaaa xxxxyyyy zzzzUUUU
- *   size = 0x10
+ * Declares a list of camera `at` points that form a spline.
+ * The points of the spline are interpolated over time to create smooth camera movements.
+ * The points are relative to the player's position and yaw.
  */
-#define CS_CAM_FOCUS_POINT_PLAYER CS_CAM_AT_REL_TO_PLAYER
-#define CS_CAM_AT_REL_TO_PLAYER(continueFlag, roll, frame, viewAngle, xPos, yPos, zPos, unused) \
-    CMD_BBH(continueFlag, roll, frame), CMD_F(viewAngle), CMD_HH(xPos, yPos), CMD_HH(zPos, unused)
+#define CS_CAM_AT_SPLINE_REL_TO_PLAYER(startFrame, endFrame) \
+    CS_CMD_CAM_AT_SPLINE_REL_TO_PLAYER, CMD_HH(0x0001, startFrame), CMD_HH(endFrame, 0x0000)
 
 /**
- * ARGS
- *   s16 unk (u), s16 startFrame (s), s16 endFrame (e)
- * FORMAT
- *   Capital U is Unused
- *   00000007 uuuussss eeeeUUUU
- *   size = 0xC
+ * Declares a single `eye` point that will be set on the specified frame, without any interpolation.
+ * Should only contain a single point, not a list.
+ * This feature is not used in the final game and lacks polish, it is recommended to use splines in all cases.
  */
-#define CS_CMD_07_LIST(unk, startFrame, endFrame, unused) \
-    CS_CMD_07, CMD_HH(unk, startFrame), CMD_HH(endFrame, unused)
+#define CS_CAM_EYE(unused0, startFrame, endFrame, unused1) \
+    CS_CMD_CAM_EYE, CMD_HH(unused0, startFrame), CMD_HH(endFrame, unused1)
 
 /**
- * ARGS
- *   s8 continueFlag (c), s8 roll (r), s16 frame (f), f32 viewAngle (a),
- *   s16 xPos (x), s16 yPos (y), s16 zPos (z)
- * FORMAT
- *   Capital U is Unused
- *   ccrrffff aaaaaaaa xxxxyyyy zzzzUUUU
- *   size = 0x10
+ * Declares a single `at` point that will be set on the specified frame, without any interpolation.
+ * Should only contain a single point, not a list.
+ * This feature is not used in the final game and lacks polish, it is recommended to use splines in all cases.
  */
-#define CS_CMD_07(continueFlag, roll, frame, viewAngle, xPos, yPos, zPos, unused) \
-    CMD_BBH(continueFlag, roll, frame), CMD_F(viewAngle), CMD_HH(xPos, yPos), CMD_HH(zPos, unused)
+#define CS_CAM_AT(unused0, startFrame, endFrame, unused1) \
+    CS_CMD_CAM_AT, CMD_HH(unused0, startFrame), CMD_HH(endFrame, unused)
```

Comment:
your comment on the CsCmdCam struct being shared by the four cam commands made me realize the macros probably should line up more idk

```suggestion
#define CS_CAM_AT(startFrame, endFrame) \
    CS_CMD_CAM_AT, CMD_HH(0, startFrame), CMD_HH(endFrame, 0)
```
note at the very least `unused` should be fixed to `unused1`